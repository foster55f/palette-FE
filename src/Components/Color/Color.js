import React from 'react';
import './Color.scss';

export const Color = (props) => {
  return (
    <span className='color-div' style={{backgroundColor: `#${props.color}`}}>
    </span>
  )
}

export default Color;