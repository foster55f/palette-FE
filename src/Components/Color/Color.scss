@import '../../scss/variables';

.color-div {
  height: 5em;
  width: 5em;
  border: 1px solid #2C2A2D;
  margin: .1em;
  border-radius: 10px;
}

