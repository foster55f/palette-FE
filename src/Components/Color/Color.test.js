import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('Color', () => {
 
it('should match the Color snapshot', () => {
    let wrapper = shallow(<Color />);
    expect(wrapper).toMatchSnapshot();
  })
})