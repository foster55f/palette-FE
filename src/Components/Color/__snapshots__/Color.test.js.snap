// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color should match the Color snapshot 1`] = `
<span
  className="color-div"
  style={
    Object {
      "backgroundColor": "#undefined",
    }
  }
/>
`;
